frontend/src/store/bucketStore.ts

```diff
@@ -44,10 +44,7 @@ export const useBucketStore = defineStore('bucket', () => {
   async function createBucketChild(parentBucketId: string, subKey: string, bucketName: string) {
     try {
       appStore.beginIndeterminateLoading();
-      const response = await bucketService.createBucketChild(parentBucketId, subKey, bucketName);
-      // push new bucket to store
-      state.buckets.value.push(response.data);
```

Comment:
maybe we should try updating the store in these 'create' actions, so we dont need to call fetchBuckets() again. the bucket table should be watching the store already. something to consider anyway